### PR TITLE
Fixed issue where checking RPM builds on aarch64 attempts to pull non-supported image (redhat/UBI7)

### PR DESCRIPTION
**Pull request by jmjaffe37**
Title says it all

### CODE REVIEW

File: linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java [hunk 1]
```diff
@@ -34,6 +34,9 @@
 public class RedHatFlavoursWithYum implements ArgumentsProvider {
 	@Override
 	public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
+		String arch = System.getProperty("os.arch");
+		boolean ubi7_supported = "x86-64".equalsIgnoreCase(arch) || "amd64".equalsIgnoreCase(arch) || "ppc64le".equalsIgnoreCase(arch) || "s390x".equalsIgnoreCase(arch);
```
Comment (gdams) [inline linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:38]: UBI doesn’t support s390x or ppc64le either

```suggestion
		boolean ubi7_supported = "x86-64".equalsIgnoreCase(arch) || "amd64".equalsIgnoreCase(arch);
```

Comment (jmjaffe37) [reply linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:38]: I may be misunderstanding the website, but are these not the same image but for different architectures?

s390x: https://catalog.redhat.com/software/containers/ubi7/ubi/5c3592dcd70cc534b3a37814?container-tabs=technical-information&architecture=s390x

ppc64cle: https://catalog.redhat.com/software/containers/ubi7/ubi/5c3592dcd70cc534b3a37814?container-tabs=technical-information&architecture=ppc64le

Comment (gdams) [reply linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:38]: You're absolutely right! Ignore me :)

**Comment by gdams** [comment]
Looks like you didn't use the same email to sign your Git commit that you did for your Eclipse Foundation account. See this guide https://adoptium.net/docs/eca-sign-off/#_how_to_change_the_author_e_mail_address_for_existing_commits to update the commit with the correct email

**Comment by gdams** [comment]
Also if you don't mind would you create the same change in the JRE directory? Currently this just fixes the tests for JDK's

File: linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java [hunk 2]
```diff
@@ -42,11 +45,19 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
...
+		builder.add(Arguments.of("oraclelinux", "7"));
+
+		/*
+		 * Redhat UBI7: Does not currently suport aarch64 architecture
+		 *     (https://catalog.redhat.com/software/containers/ubi7/ubi/5c3592dcd70cc534b3a37814?container-tabs=technical-information).
+		 */
+		if (ubi7_supported) {
+			builder.add(Arguments.of("registry.access.redhat.com/ubi7/ubi", "latest"));
+		}
+
```
Comment (karianna) [inline linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:59]:
```suggestion
```

**Comment by jmjaffe37** [comment]
PR is now signed correctly :)

File: linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java (hunk 1 above)
Comment (jmjaffe37) [inline linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:38]: Can someone confirm that this is a correct value of `System.getProperty("os.arch")`, and not "x86_64"? I wasn't sure which of these to put based on what I found on the internet

Comment (karianna) [reply linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:38]: In Java, System.getProperty("os.arch") returns a string representing the underlying system's architecture. For machines running on x86 64-bit architectures, the typical return value is "amd64" or "x86_64" depending on the specific JVM implementation. However, "amd64" is the more common return value. Note that the exact return value may vary between different JVM vendors and versions.

**Review by karianna — APPROVED** [review]

File: linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java (hunk 1 above)
Comment (jmjaffe37) [reply linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java]: I just pushed a new change to fix this. It gets rid of non-alphanumeric characters before comparing the arch to predetermined values

**Review by gdams — APPROVED** [review]

**Pull request merged**